### PR TITLE
test: add local setup for docs e2e tests [SFUI2-933]

### PR DESCRIPTION
**Pull request by Szymon-dziewonski**
# Related issue

https://vsf.atlassian.net/browse/SFUI2-933

# Scope of work

Adding turbo workspace package `@storefront-ui/docs-tests` that can be run in pair with other processes. 
Add readme how to run cypress e2e test for docs locally without needing run everything manually

# Screenshots of visual changes

# Checklist

- [x] Self code-reviewed
- [ ] Changes documented
- [ ] Semantic HTML
- [ ] SSR-friendly
- [ ] Caching friendly
- [ ] a11y for WCAG 2.0 AA
- [ ] examples created
- [ ] blocks created
- [ ] cypress tests created

### CODE REVIEW

File: apps/docs/development/testing.md [hunk 1]
```diff
@@ -9,11 +9,31 @@ We run every test file against 2 different framework environments:
...
 In opposition to running them simultaneously, framework test suites can also be run one-by-one. To do that navigate to either `apps/preview/next` (for `React` test suite) or `apps/preview/vue` (for `Vue` test suite) and run `yarn test` command.
 
+### E2E testing of documentation
+
+In order to run whole process:
+
+1. run development Nuxtjs so it would be available through http://localhost:3001
+2. run development Nextjs so it would be available through http://localhost:3002
+3. run documentation so it would be accessible through http://localhost:8080/v2
+4. wait for all above
```
Comment (FRSgit) [inline apps/docs/development/testing.md:26]: I think you also need to open up these at least once to get the initial dev build done, right?

File: apps/docs/tests/cypress.config.ts [hunk 2]
```diff
@@ -6,11 +6,10 @@ export default defineConfig({
...
   defaultCommandTimeout: 20000,
   video: false,
-  
   e2e: {
     baseUrl: 'http://localhost:8080/v2',
     specPattern: '../tests/**/*.cy.{js,jsx,ts,tsx}',
     supportFile: '../tests/support/index.ts',
   },
-  chromeWebSecurity: false
+  chromeWebSecurity: false,
```
Comment (FRSgit) [inline apps/docs/tests/cypress.config.ts:14]: Is that needed?

**Review by FRSgit — APPROVED** [review]

File: apps/docs/tests/cypress.config.ts (hunk 2 above)
Comment (Szymon-dziewonski) [reply apps/docs/tests/cypress.config.ts:14]: auto formatting after removal empty line, so not required but welcomed :)

Comment (FRSgit) [reply apps/docs/tests/cypress.config.ts:14]: I meant `chromeWebSecurity: false`

Comment (Szymon-dziewonski) [reply apps/docs/tests/cypress.config.ts:14]: This is soemthing I wasnt done myself but I will check it withing CI setup

**Pull request merged**